rgw/v2/tests/s3_swift/test_policy_torrent.py

```diff
@@ -91,6 +91,10 @@ def test_exec(config, ssh_con):
         rgw_tenant1_user1,
         tenant1_user1_info,
     )
+    rgw_service_name = utils.exec_shell_cmd("ceph orch ls | grep rgw").split(" ")[0]
+    utils.exec_shell_cmd(
+        f"ceph config set client.{rgw_service_name} rgw_torrent_flag true"
+    )
```

Comment:
@TejasC88 
don't we need rgw restart after this?

Reply:
@hmaheswa I had missed this , but somehow the config is taken without a restart also . Let me add a restart just in case , thanks.